### PR TITLE
Feature/mentee mentor intake location questions

### PR DESCRIPTION
**Pull request by brettsmith212**
## Description

Mentee and Mentor Intake Forms have been updated with the correct location question formatting. Now you can select a country from a list of countries. If United States is selected a state dropdown bar appears for the user to select a state as well as a free text for a city. Otherwise, only a city is required.

@amarrazza helped with pair programming on this PR.

#### Video Link

https://www.loom.com/share/3bfe81103cc746a09ac70b50bd87279b

#### Trello Link

<blockquote class="trello-card"><a href="https://trello.com/c/FwvJEuAu/345-mentee-and-mentor-intake-location-questions">Feature: Update Mentor Mentee Intake Form to have new Location Questions format</a></blockquote>

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have removed unnecessary comments/console logs from my code
- [x] My changes generate no new warnings
- [x] I have checked my code and corrected any misspellings
- [x] No duplicate code left within changed files
- [x] Size of pull request kept to a minimum
- [x] Pull request description clearly describes changes made & motivations for said changes

### CODE REVIEW

**Review by VABIII — APPROVED** [review]
Nice job updating the input fields to meet customer specifications. The video explanation and walkthrough were done quite well also.

**Review by mikayla-kosmala — CHANGES_REQUESTED** [review]
Nice job at going above and beyond by making sure that the mentor intake form also was collecting the data in the database! 

Only request is to make sure it passes the second test!

**Comment by brettsmith212** [comment]
> 

@paulstgermain any thoughts on the reason why many of the PR's are not passing the second test?

**Comment by brettsmith212** [comment]
> 

@mikayla-kosmala I think I found the issue with the tests. 3 of them are referencing pages that no longer exist. I commented them out and made a not for the future to recreate these tests.  Now the code is passing all tests.

**Review by mikayla-kosmala — APPROVED** [review]
Nice job at finding why all the PRs are failing!

File: src/components/pages/Navbar/Navbar.js [hunk 1]
```diff
@@ -12,7 +12,6 @@ import { useOktaAuth } from '@okta/okta-react';
 import { getProfile } from '../../../state/actions/userProfile/getProfile';
 import LoginButton from './NavbarFeatures/LoginButton';
 import SignupButton from './NavbarFeatures/SignupButton';
-import { useAuth0 } from '@auth0/auth0-react';
```
Comment (paulstgermain) [inline src/components/pages/Navbar/Navbar.js:15]: Let's reintroduce this line and comment it out if it's causing problems - we'll be needing this soon!

File: src/components/pages/Navbar/NavbarFeatures/LoginButton.js [hunk 2]
```diff
@@ -1,6 +1,5 @@
 import React from 'react';
 import { NavBtnLink } from '../../NavBarLanding/NavBarStyle';
-import { useAuth0 } from '@auth0/auth0-react';
```
Comment (paulstgermain) [inline src/components/pages/Navbar/NavbarFeatures/LoginButton.js:3]: Let's bring this back too!

**Review by paulstgermain — CHANGES_REQUESTED** [review]
Great work here - and excellent job fixing our failing auto-tests! Just one small change - let's bring back those Auth0 lines you deleted. We'll be needing them soon - we can simply leave them commented out for now.

File: src/components/pages/Navbar/Navbar.js (hunk 1 above)
Comment (brettsmith212) [reply src/components/pages/Navbar/Navbar.js]: Added

File: src/components/pages/Navbar/NavbarFeatures/LoginButton.js (hunk 2 above)
Comment (brettsmith212) [reply src/components/pages/Navbar/NavbarFeatures/LoginButton.js]: Added. Thanks for the heads up!

**Review by paulstgermain — APPROVED** [review]

**Pull request merged**